pkg/failovercluster/storage/clustersharedvolume/clustersharedvolume_test.go

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+package clustersharedvolume
```

Comment:
We need a way to test in windows HCI setup

Reply:
We can use 1es to test these.